4.6 Improve OpenShift Logging monitoring (#25159)
openshift/openshift-docs: modules/cluster-logging-elasticsearch-rules.adoc

|Description
|Severity

|ElasticsearchBulkRequestsRejectionJumps

> |Elasticsearch is experiencing an increase in bulk rejections on the specified node. This node may not be keeping up with the indexing speed."

Reviewer: ```suggestion
|Elasticsearch is experiencing an increase in bulk rejections on the specified node. This node might not be keeping up with the indexing speed.
```

|Warning

|ElasticsearchClusterNotHealthy

> |The cluster health status has been RED for at least 2m. The cluster does not accept writes, shards may be missing, or themaster

Reviewer 2: `themaster` missing space.

 node hasn't been elected yet.
|critical

|ElasticsearchClusterNotHealthy

> |The cluster health status has been YELLOW for at least 20m. Some shard replicas are not allocated.

Reviewer: ```suggestion
|The cluster health status has been YELLOW for at least 20 minutes. Some shard replicas are not allocated.
```

|warning
 
|ElasticsearchDiskSpaceRunningLow
|The cluster is expected to be out of disk space within the next 6 hours.
|Critical

|ElasticsearchHighFileDescriptorUsage

> |The cluster is predicted to be out of file descriptors within the next hour

Reviewer: ```suggestion
|The cluster is predicted to be out of file descriptors within the next hour.
```

|warning

|ElasticsearchJVMHeapUseHigh
|The JVM Heap usage on the specified node is high.
|Alert

|ElasticsearchNodeDiskWatermarkReached

> |The specified node has hit the low watermark due to low free disk space. Shards can not be allocated to this node anymore. You should

Reviewer: ```suggestion
|The specified node has hit the low watermark due to low free disk space. Shards can not be allocated to this node anymore.
```
Also, consider using a different term than watermark in both this entry and the next.

> consider adding more disk space to the node.

Reviewer: ```suggestion
Consider adding more disk space to the node.
```

|info

|ElasticsearchNodeDiskWatermarkReached
|The specified node has hit the high watermark due to low free disk space. Some shards will be re-allocated to different
nodes if possible. Make sure more disk space is added to the node or drop old indices allocated to this node.
|warning

|ElasticsearchNodeDiskWatermarkReached

> |The specified node has hit the flood watermark due to low free disk space. Every index having a shard allocated on this node is enforced a read-only block. The index block must be manually released when the disk utilization falls below the high watermark.

Reviewer: ```suggestion
|The specified node has hit the flood watermark due to low free disk space. Every index that has a shard allocated on this node is enforced a read-only block. The index block must be manually released when the disk use falls below the high watermark.
```

|critical

> ?|ElasticsearchJVMHeapUseHigh

Reviewer: Is the ? intentional?
